charts/qiskit-serverless/charts/gateway/values.yaml

```diff
@@ -145,17 +145,21 @@ ingress:
   #    hosts:
   #      - chart-example.local
 
-resources: {}
-  # We usually recommend not to specify default resources and to leave this as a conscious
-  # choice for the user. This also increases chances charts run on environments with little
-  # resources, such as Minikube. If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  # limits:
-  #   cpu: 100m
-  #   memory: 128Mi
-  # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+resources:
+  limits:
+    cpu: "1500m"
+    memory: "1Gi"
+  requests:
+    cpu: "500m"
+    memory: "700Mi"
+scheduler:
+  resources:
+    limits:
+      cpu: "500m"
```

Comment:
From what I was seeing in the metrics it seems `scheduler` is using more `cpu` (like 3 cores), `memory` seems fine.

Reply:
OK.  It is working in my system but I'll update to 1500m.